Cargo.toml

```diff
@@ -54,6 +54,8 @@ httpmock = "0.6"
 mockito = "1.2.0"
 pretty_assertions = "1.4.0"
 stripmargin = "0.1.1"
+reqwest = "0.11"
+serde_json = "1.0"
```

Comment:
They are already dependencies of the project.

Reply:
i have updated the dependencies check it once thank you